cmd/manager/main.go

```diff
@@ -266,6 +271,9 @@ func Command() *cobra.Command {
 	_ = cmd.Flags().MarkHidden(operator.AutoPortForwardFlag)
 	_ = cmd.Flags().MarkHidden(operator.DebugHTTPListenFlag)
 
+	// hide the flag used for E2E test only
+	_ = cmd.Flags().MarkHidden(operator.TelemetryIntervalFlag)
```

Comment:
I'm wondering if this is necessary. I don't see a particular case where users would like to tweak this, but I also don't see any problem in having it exposed so we keep all flags explicit and there's no hidden secret?

Reply:
My motivation was that we already know the right value for it and if not for the E2E test, this wouldn't be configurable at all, just another constant. I also don't see a use case for changing it (there is already a flag to disable it completely), so I don't think there is a compelling reason to add it to the list of configuration options that users need to read/know about/understand.

Reply:
I'm fine with keeping it hidden.